src/main/java/uk/gov/hmcts/reform/roleassignmentbatch/RoleAssignmentBatchApplication.java

```diff
@@ -14,8 +14,8 @@
 
     public static void main(String[] args) throws Exception {
         final ApplicationContext context = SpringApplication.run(RoleAssignmentBatchApplication.class, args);
-        log.info("Putting application to sleep for 5 mins");
-        Thread.sleep(1000 * 60 * 5);
+        log.info("Putting application to sleep");
+        Thread.sleep(1000 * 5L);
         log.info("The sleep is complete.");
         int exitCode = SpringApplication.exit(context);
```

Comment:
why do we need this sleep thread? don't we need to define @EnableBatchProcessing annotation